lte/gateway/python/scripts/subscriber_cli.py

```diff
@@ -96,17 +96,21 @@ def update_subscriber(client, args):
         pre_vul = "preemption_vulnerability"
         ul = "mbr_uplink"
         dl = "mbr_downlink"
-        static_ip = "static_ip"
-        vlan_id = "vlan"
-        gw_ip = "gw_ip"
-        gw_mac = "gw_mac"
-
-        apn_keys = (apn_name, qci, priority, pre_cap, pre_vul, ul, dl,
-                    static_ip, vlan_id, gw_ip, gw_mac)
```

Comment:
why did you remove these other fields?

Reply:
I think it was overwritten while merging. Will correct it